src/EUPL-1.0.xml

```diff
@@ -0,0 +1,249 @@
+          <p>In case of any Distribution and/or Communication of the Work by means of electronic communication
+             by You (for example, by offering to download the Work from a remote location) the distribution
+             channel or media (for example, a website) must at least provide to the public the information
+             requested by the applicable law regarding the identification and address of the Licensor, the
+             Licence and the way it may be accessible, concluded, stored and reproduced by the
+             Licensee.</p>
+        </li>
+        <li>
+          <b>12.</b>
+          <p>Termination of the Licence</p>
+          <p>The Licence and the rights granted hereunder will terminate automatically upon any breach by the
+             Licensee of the terms of the Licence.</p>
+          <p>Such a termination will not terminate the licences of any person who has received the Work from
+             the Licensee under the Licence, provided such persons remain in full compliance with the
+             Licence.</p>
+        </li>
+        <li>
+          <b>13.</b>
+          <p>Miscellaneous</p>
+          <p>Without prejudice of Article 9 above, the Licence represents the complete agreement between the
+             Parties as to the Work licensed hereunder.</p>
+          <p>If any provision of the Licence is invalid or unenforceable under applicable law, this will not
+             affect the validity or enforceability of the Licence as a whole. Such provision will be
+             construed and/or reformed so as necessary to make it valid and enforceable.</p>
+          <p>The European Commission may put into force translations and/or binding new versions of this
+             Licence, so far this is required and reasonable. New versions of the Licence will be published
+             with a unique version number. The new version of the Licence becomes binding for You as soon
+             as You become aware of its publication.</p>
+        </li>
+        <li>
+          <b>14.</b>
+          <p>Jurisdiction</p>
+          <p>Any litigation resulting from the interpretation of this License, arising between the European
+             Commission, as a Licensor, and any Licensee, will be subject to the jurisdiction of the Court
+             of Justice of the European Communities, as laid down in article 238 of the Treaty establishing
+             the European Community.</p>
+          <p>Any litigation arising between Parties, other than the European Commission, and resulting from
+             the interpretation of this License, will be subject to the exclusive jurisdiction of the
+             competent court where the Licensor resides or conducts its primary business.</p>
+        </li>
+        <li>
+          <b>15.</b>
+          <p>Applicable Law</p>
+        </li>
+      </list>
+      <p>This Licence shall be governed by the law of the European Union country where the Licensor resides or has
```

Comment:
These paragraphs need to be made part of bullet 15 above. Pull request submitted.